DEVELOPMENT.md

```diff
@@ -4,12 +4,13 @@
 This is the simplest configuration for developers to start with.
 
 ### Initial Setup
-1. Run `docker compose run --rm django ./manage.py migrate`
-2. Run `docker compose run --rm django ./manage.py createcachetable`
-3. Run `docker compose run --rm django ./manage.py createsuperuser --email $(git config user.email)`
+1. Run `docker compose build --build-arg USERID=$(id -u) --build-arg GROUPID=$(id -g) --build-arg LOGIN=$(id -n -u)` to build the development container image. This builds the image to work with your (non-root) development user so that the linting and formatting commands work inside and outside of the container. If you prefer to build the container image so that it runs as `root`, you can omit the `--build-arg` arguments (but you will likely run into trouble running those commands).
+2. Run `docker compose run --rm django ./manage.py migrate`
```

Comment:
note, it might be worth just to number them all as `1.` to avoid causing the diff if only indexing to change. Then numbers would be rendered by the markdown renderer

Reply:
It is a nice feature of Markdown but I'm going to avoid it because for the most part, people are reading the text version of this file in their dev environment, rather than the rendered version that appears on GitHub (meaning, numbering the steps explicitly is necessary).